src/notifications/api/tests/test_abonnement.py

```diff
@@ -0,0 +1,142 @@
+from django.test import override_settings
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+from vng_api_common.tests import JWTScopesMixin, get_operation_url
+
+from notifications.datamodel.models import (
+    Abonnement, Filter, FilterGroup, Kanaal
+)
+from notifications.datamodel.tests.factories import (
+    AbonnementFactory, KanaalFactory
+)
+
+from ..scopes import SCOPE_NOTIF_CHANGE_ALL, SCOPE_NOTIF_READ_ALL
+
+
+@override_settings(
+    LINK_FETCHER='vng_api_common.mocks.link_fetcher_200',
+    ZDS_CLIENT_CLASS='vng_api_common.mocks.MockClient'
+)
+class AbonnementenTests(JWTScopesMixin, APITestCase):
+
+    scopes = [
+        SCOPE_NOTIF_CHANGE_ALL,
+        SCOPE_NOTIF_READ_ALL,
+    ]
+
+    def test_abonnementen_create(self):
+        """
+        test /abonnementen POST:
+        create abonnement with nested kanalen and nested filters via POST request
+        check if data were parsed to models correctly
+        """
+
+        kanaal_zaken = KanaalFactory.create(naam='zaken')
+        kanaal_informatieobjecten = KanaalFactory.create(naam='informatieobjecten')
+        abonnement_create_url = get_operation_url('abonnement_create')
+        data = {
+            "callbackUrl": "https://ref.tst.vng.cloud/zrc/api/v1/callbacks",
+            "auth": "Bearer aef34gh",
+            "kanalen": [{
+                "naam": "zaken",
+                "filters": [
+                    {"bron": "082096752011"},
+                    {"zaaktype": "example.com/api/v1/zaaktypen/5aa5c"},
+                    {"vertrouwelijkeidaanduiding": "*"}
```

Comment:
Can you search replace `vertrouwelijkeidaanduiding` with `vertrouwelijkheidaanduiding` :)